README.md

```diff
@@ -122,8 +122,32 @@ brew install ugit
 
 #### Windows
 
-Using ugit on windows is still in experimental phases. You can use the new windows terminal + git bash to run ugit. Please read [this thread for more info](https://github.com/Bhupesh-V/ugit/issues/20#issuecomment-1048234725)
+To use ugit on windows you need to install:<br>
+* Git Bash<br>
+Git bash comes with git (if you have installed it on your machine then no need).<br>
+To install git (preferably with scoop):
+```
+scoop install git
+```
+* FZF  
+```
+scoop install fzf
+``` 
+Alternatively, with Chocolatey:
+```
+choco install fzf
+```
```

Comment:
Indent these code blocks to same level of list. If you are using VSCode please use the markdown preview feature

Edit the instructions like this
![de](https://user-images.githubusercontent.com/34342551/186604149-bcc1bb28-c888-48c1-8d3a-4ef2a6a04536.png)